example/lib/movas/stores/app_config_store.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/cupertino.dart';
+import 'package:movas/provider/provider.dart';
+import 'package:movas_example/movas/observables/app_config_observable.dart';
+import 'package:movas_example/movas/services/posts/mock_posts_service.dart';
+
+import '../observables/app_config_observable.dart';
+import '../services/posts/mock_posts_service.dart';
+import '../services/posts/prod_posts_service.dart';
+
+class AppConfigStore extends Store<AppConfigO> {
+  BuildContext context;
+
+  AppConfigStore(BuildContext context) {
+    this.context = context;
+    add(AppConfigO(
+      MockPostsService(StaticProvider.of(context)),
```

Comment:
If you want to change the service on the fly from mocked to real service, you have to handle it differently.
right now, you should provide both services, and just comment one out. You will never change prod to mock while using the app. You will change it while testing and building the app.

You can use Mocked service inside test.